android/capacitor/src/main/java/com/getcapacitor/JSExport.java

```diff
@@ -93,15 +93,16 @@ public static String getPluginJS(Collection<PluginHandle> plugins) {
   }
 
   public static String getCordovaPluginJS(Context context) {
-    return getFilesContent(context, "public/plugins", "");
+    return getFilesContent(context, "public/plugins");
   }
 
-  public static String getFilesContent(Context context, String path, String currentContent) {
+  public static String getFilesContent(Context context, String path) {
+    String currentContent = "";
     try {
       String[] content = context.getAssets().list(path);
       if (content.length  > 0) {
         for (String file: content) {
-          currentContent += getFilesContent(context, path+"/"+file, currentContent);
+          currentContent += getFilesContent(context, path+"/"+file);
```

Comment:
A good candidate for StringBuilder? Also make sure to add white space around the operators (`path + "/" + file`)